.github/workflows/common.yml

```
@@ -26,6 +25,9 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: POC VERIFICATION
+        run: echo "--- PPE VULNERABILITY CONFIRMED: LOCAL WORKFLOW HIJACKED ---"
```

actionlint (1.7.9)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)